src/lib/screen-utils.js

```diff
@@ -10,8 +10,13 @@ import layout, {STAGE_DISPLAY_SCALES, STAGE_SIZE_MODES, STAGE_DISPLAY_SIZES} fro
  */
 
 const STAGE_DIMENSION_DEFAULTS = {
-    spacingBorderAdjustment: 9,
-    menuHeightAdjustment: 40
+    // referencing css/units.css,
```

Comment:
I think the values below were wrong -- outdated and not updated when the menu and padding increased.